modules/pubmaticAnalyticsAdapter.js

```diff
@@ -407,7 +417,7 @@ function executeBidsLoggerCall(e, highestCpmBids) {
       'au': origAdUnit.owAdUnitId || getGptSlotInfoForAdUnitCode(adUnitId)?.gptSlot || adUnitId,
       'mt': getAdUnitAdFormats(origAdUnit),
       'sz': getSizesForAdUnit(adUnit, adUnitId),
-      'ps': gatherPartnerBidsForAdUnitForLogger(adUnit, adUnitId, highestCpmBids.filter(bid => bid.adUnitCode === adUnitId)),
+      'ps': gatherPartnerBidsForAdUnitForLogger(adUnit, adUnitId, highestCpmBids.filter(bid => bid.adUnitCode === adUnitId), e),
```

Comment:
can you also fix this? https://github.com/prebid/Prebid.js/blob/4fc7f8865e3f6971b8d880f662a4ac03e05a7109/modules/pubmaticAnalyticsAdapter.js#L404 

'$prebid.version$' works, you do not need to import the global for it